### DISCUSSION TITLE
getActivePinia was called with no active Pinia.Did you forget to install pinia?

### DISCUSSION THREAD

**Discussion by Linhongbin-1111** [vuejs/pinia#1323] [Help and Questions] [answered]
I got this error when i using useStore() in the router. Following the official website example, I used store in router.beforeEach(), but the problem has not been solved.

**Comment by posva** [comment, marked as answer]
are you installing the pinia plugin **before** the router?

```ts
app.use(pinia).use(router)
```

**Reply by Linhongbin-1111** [reply]
Thanks for your reply. I found my problem. https://github.com/vuejs/pinia/discussions/1238#discussioncomment-2832485
I'm sorry for the inconvenience.Thanks again.

**Comment by Linhongbin-1111** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我是林鸿斌。我已收到你的邮件，谢谢！
